doc/tutorials/cloud/00_setup.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        if jar_exists:
</pre><p>we probably need an overwrite parameter for that, it happens that uploaded files are broken.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>In principle we can get the checksum from artefacts and do validation of the file. But I fought it would be too complex for the prototype (and will add overhead, as jar file is large - 500MB)</p>

Reply:
<p>But I can add the argument to enforce the reloading, that's not a problem</p>

Reply:
<p>yep, flag to redownload the file and also reupload to the BucketFs should be fine. We might replace that in the long term anyway with the extension manager.</p>